src/games/strategy/triplea/ui/TripleAMenu.java

```diff
@@ -988,8 +988,8 @@ private void createAndSaveStats(final boolean showPhaseStats) {
       }
       text.append("\n");
       clone.getHistory().gotoNode(clone.getHistory().getLastNode());
-      @SuppressWarnings("rawtypes")
-      final Enumeration nodes = ((DefaultMutableTreeNode) clone.getHistory().getRoot()).preorderEnumeration();
+      @SuppressWarnings("unchecked")
+      final Enumeration<HistoryNode> nodes = ((DefaultMutableTreeNode) clone.getHistory().getRoot()).preorderEnumeration();
```

Comment:
@RoiEXLab The casting later on in the code is now no longer necessary:
Line 997: `        final HistoryNode element = (HistoryNode) nodes.nextElement();`